CentralManagement/CentralManagementSetupWeb.md

```diff
@@ -247,6 +208,22 @@ You should received a notification similar to this:
 
 ![Test email sent successfully](images/features/ccm/test_email_sent_correctly.png)
 
+### Step 4.3: LDAP Configuration
+
+CCM is able to bind with LDAP to enable authentication to Central Management using an LDAP user account and password.
+
+1. Open the CCM Site in the browser
+1. Login with the `ccmadmin` user
+1. In the left hand menu click on `Administration` and then `Settings`
+1. Click on the `User management` tab in the `Settings` screen
+1. Under LDAP Setting click the `Enable LDAP Authentication` button
+1. Fill in your FQDN for the `Domain name` field
+1. Fill in the `User name` field with an active directory account that has access to query user accounts within your active directory enviornment.
+1. Fill the `Password` field with the password for the active driectory user name used above
+1. Click the `Save All` button at the top right of the page to save your settings.

```

Comment:
Be consistent with periods or not at the end of each list item.

Reply:
Added periods to end of all list items for document continuity.